tests/jballerina-unit-test/src/test/resources/test-src/structs/object-function-code-analysis.bal

```diff
@@ -0,0 +1,6 @@
+function printDetails(string name,
+                      int age = 18,
+                      string... modules) {
+    string detailString = "Name: " + name + ", Age: " + age.toString();
+
+}
```

Comment:
Add newline at the end of file.

Reply:
Yeah but I can see other files at the location contains a newline at the EOF. For example: [invalid-struct-literal-key-negative.bal](https://github.com/ballerina-platform/ballerina-lang/blob/master/tests/jballerina-unit-test/src/test/resources/test-src/structs/invalid-struct-literal-key-negative.bal)

Reply:
@deshankoswatte I can't see any newlines at the EOF.

![image](https://user-images.githubusercontent.com/16335290/78453455-74411c80-76af-11ea-8d90-8b03f95b59a8.png)

Reply:
> @deshankoswatte I can't see any newlines at the EOF.

You can't see the new line when you look through github and it is visible when you open it from an IDE.

@gigara if you don't have a newline at the end of the file github indicates it with a warning as you see at the end of the left side file(you can see the same at the end of the file `object-function-code-analysis.bal` which is added by this PR) :
![Screenshot from 2020-04-04 21-30-57](https://user-images.githubusercontent.com/33062368/78455377-a5274e80-76bb-11ea-93d8-8ee165c114e6.png) And again the same file on the right side of the screenshot there you have added the newline by the [PR](https://github.com/ballerina-platform/ballerina-lang/pull/22444/files) so because of that you can't see the warning there.

Reply:
@deshankoswatte Here I've attached the IDE preview of the above you mentioned file. I can't find any new line in there also. Anyway, I'll fix it. Thanks

![image](https://user-images.githubusercontent.com/16335290/78455902-78286b00-76be-11ea-99a4-f38bdc642d20.png)

Reply:
All,

In the formatting there are two file types... first type the .bal files which gonna be formatted in the test and second type is the expected out put.  So in this case we remove the new line from the letExpr.bal file as the formatter should add the new line at the EOF.... anyway in other files its better to add new line at the EOF...